src/components/Common/Accordion/index.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { useState } from 'react';
+import AnimateHeight from 'react-animate-height';
+
+export interface AccordionProps {
+  children: (args: AccordionChildProps) => React.ReactElement<any, any> | null;
+  /** If true, only one accordion item can be open at any time */
+  single?: boolean;
+  /** If true, at least one accordion item will always be open */
+  atLeastOne?: boolean;
+  initialOpenIndexes?: number[];
+}
+export interface AccordionChildProps {
+  openIndexes: number[];
+  handleClick(index: number): void;
+  AccordionContent: any;
+}
+
+export const AccordionContent: React.FC<{ isOpen: boolean }> = ({
+  isOpen,
+  children,
+}) => {
+  return (
+    <AnimateHeight height={isOpen ? 'auto' : 0}>
+      {children || <></>}
```

Comment:
Can we not just have this as the following?

```suggestion
      {children}
```